src/Orc.NuGetExplorer.Xaml/Orc.NuGetExplorer.Xaml.Shared/Extensions/StringExtensions.cs

```diff
@@ -8,13 +8,18 @@
 namespace Orc.NuGetExplorer
 {
     using System.Windows.Documents;
+    using System.Windows.Media;
 
     public static class StringExtensions
     {
         #region Methods
-        public static Inline ToInline(this string text)
+        public static Inline ToInline(this string text, Brush brush = null)
```

Comment:
This is a breaking change. Next time please mark this one as obsolete and introduce a new one.